tests/taipy/gui/e2e/test_timezone.py

```diff
@@ -65,3 +65,18 @@ def _timezone_test_template(page: "Page", gui: Gui, helpers, time_zone, texts):
     page.wait_for_selector("#text1")
     text1 = page.query_selector("#text1")
     assert text1.inner_text() in texts
+
+
+def test_date_only(page: "Page", gui: Gui, helpers):
+    page_md = """
+<|{t}|id=text1|>
+"""
+    t = _ISO_to_date("Wed Jul 28 1993")  # noqa: F841
```

Comment:
That is not ISO... or the function is wrongly named, and ENG-locale dependent... which is bad